proxy/src/auth/backend/console_redirect.rs

```diff
@@ -157,6 +157,15 @@ async fn authenticate(
         }
     }
 
+    // Check if the access over the public internet is allowed, otherwise block. Note that
+    // the console redirect is not behind the VPC service endpoint, so we don't need to check
+    // the VPC endpoint ID.
+    if let Some(public_access_allowed) = db_info.public_access_allowed {
```

Comment:
Not necessary, but we can use this as well:

`account_id.map(|id| self.insert_account2endpoint(id, endpoint_id));`

Reply:
Not sure I understand the suggestion... Can you elaborate (here or on Slack)?

Reply:
Oh, I pointed out another function from a different place here :)  I think that's fine, nevermind